packages/core/src/zip/__tests__/walk-benchmark.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * @jest-environment node
+ */
+
+import { Suite } from 'benchmark'
+import I from 'immutable'
+import R from 'ramda'
+
+import * as zip from '..'
+import * as data from '../../data'
+
+import { postWalk as zippaPostWalk } from '../../vendor/zippa/walk'
+
+import navigation from './fixture/navigation.json'
+
+describe('elementZipper walking benchmarks', () => {
+  const state = I.fromJS(navigation)
+
+  const update = cmsIds => el => {
+    if (
+      data.isOfKind(['component', 'action-buttons'], el) &&
+      cmsIds.includes(el.get('cmsId'))
+    ) {
+      return el.set('bookmarked', true)
+    }
+    return el
+  }
+
+  test('alternative walking strategies', () => {
+    const testWithPostWalkFromZippa = cmsIds =>
+      R.pipe(
+        zip.elementZipper({}),
+        zippaPostWalk(update(cmsIds)),
+        zip.value
```

Comment:
When all methods are implemented in skele's `zipper.js`, we need to use version from zippa here.